common/src/main/java/com/wynntils/overlays/custombars/CustomBarOverlayBase.java

```diff
@@ -19,7 +19,7 @@ public abstract class CustomBarOverlayBase extends BarOverlay {
     public final Config<String> valueTemplate = new Config<>("");
 
     @Persisted(i18nKey = "feature.wynntils.customBarsOverlay.overlay.customBarBase.enabledTemplate")
-    public final Config<String> enabledTemplate = new Config<>("string_equals(world_state;\"WORLD\")");
```

Comment:
You should change the `isRendered` method here, the same way you did for `TextOverlay`